beater/beater.go

```diff
@@ -549,7 +549,8 @@ func (s *serverRunner) run(listener net.Listener) error {
 	}
 	runServer = s.wrapRunServerWithPreprocessors(runServer)
 
-	batchProcessor := make(modelprocessor.Chained, 0, 3)
+	batchProcessor := make(modelprocessor.Chained, 0, 4)
```

Comment:
```suggestion
	var batchProcessor modelprocessor.Chained
```

this is not a hot code path, let's simplify a little